docs/pre-release/howtos/intercepts.md

```diff
@@ -278,41 +277,45 @@ will be routed to your cluster as usual.
 
    ```console
    $ telepresence intercept example-service --port 8080:http --env-file ~/example-service-intercept.env
-   To create a preview URL, telepresence needs to know how cluster
-   ingress works for this service.  Please Confirm the ingress to use.
+      To create a preview URL, telepresence needs to know how requests enter 
+	    your cluster. Please Select the ingress to use.
```

Comment:
I'm assuming it's just because `Confirm` was in the prior reason, but could we make an issue in https://github.com/datawire/telepresence2-proprietary to fix the capitalization? we're just under a crunch to get this out so I'd like to not have to make edits + build a new rc + test it.  That being said if you think it's blocking the release that's okay!

Reply:
Ah, I see.
Yeah, that's fine. I'll create an issue shortly. No need to hold things up.